Tests/SwinjectAutoregistrationTests/AutoregistrationSpec.swift

```diff
@@ -209,6 +209,7 @@ class AutoregistrationSpec: QuickSpec {
             }
             
             #if !os(Linux)
+            #if !SWIFT_PACKAGE
```

Comment:
We can remove `!os(Linux)` condition - `!SWIFT_PACKAGE` is stronger & more fitting